### ISSUE TITLE
add page summarising libraries/gems available

### ISSUE THREAD

**Issue by davewhiteland** [everypolitician/everypolitician-docs#23]
Because we've made (and are still making) it supereasy for devs to use the data, but not made it supereasy for them to know this is the case.

Simples. Increasingly urgent.

**Comment by tmtmtmtm** [comment]
Yep. We still need to do a lot of work on those libraries (and port them to other languages) but the fact you can already do things like 

``` ruby
require 'everypolitician'
puts EveryPolitician.legislature('UK', 'Commons').popolo.current_term.start_date
```

and have it Just Work, is something we should be making much much more obvious.
